iothub/service/src/Common/Extensions/CommonExtensions.cs

```diff
@@ -194,19 +80,9 @@ public static void AppendKeyValuePairIfNotEmpty(this StringBuilder builder, stri
         /// </summary>
         /// <param name="value">The value to check.</param>
         /// <returns>Returns true if the value is null or empty, otherwise returns false.</returns>
-        public static bool IsNullOrWhiteSpace(this string value)
+        internal static bool IsNullOrWhiteSpace(this string value)
```

Comment:
Kind of a dumb helper method. We should just natively use `stirng.IsNullOrWhiteSpace` instead.